src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -156,7 +156,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
+                                _logger.LogInformation(0, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
```

Comment:
This warning could very well be actionable if you're not running in a shared hosting environment.